website/web/__init__.py

```diff
@@ -660,19 +660,31 @@ def historical_lookups(tree_uuid: str) -> str | WerkzeugResponse | Response:
                            circl_pdns=data.get('circl_pdns'))
 
 
-@app.route('/tree/<string:tree_uuid>/categories_capture/', defaults={'query': ''})
+@app.route('/tree/<string:tree_uuid>/categories_capture/', defaults={'query': ''}, methods=['GET', 'POST'])
 @app.route('/tree/<string:tree_uuid>/categories_capture/<string:query>', methods=['GET'])
 def categories_capture(tree_uuid: str, query: str) -> str | WerkzeugResponse | Response:
     if not enable_categorization:
         return redirect(url_for('tree', tree_uuid=tree_uuid))
-    current_categories = lookyloo.categories_capture(tree_uuid)
     matching_categories = None
-    if query:
+    if 'verification-status' in request.form:
+        # fast categories
+        categories = []
+        possible_ctgs = {'legitime': ["parking-page", "default-page", 'institution', 'captcha', 'authentication-form', 'adult-content', 'shop'],
```

Comment:
Say "legitimate" instead of "legitime" (I think?)

Reply:
True, I've always liked inventing words

Reply:
That's also the french one ;)